src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -1920,7 +1920,15 @@ IEnumerable<ITableIndex> GetIndexesToRebuild()
 
     #region System column utilities
 
-    private bool IsSystemColumn(string name) => SystemColumnNames.Contains(name);
+    private bool IsSystemColumn(string name)
```

Comment:
nit: this can use a conditional operator and still be an expression-bodied method